15-greenhouse-monitor/firmware/greenhouse_old/app.h

```diff
@@ -0,0 +1,85 @@
+// Copyright 2023 Blues Inc.  All rights reserved.
+// Use of this source code is governed by licenses granted by the
+// copyright holder including that found in the LICENSE file.
+
+#pragma once
```

Comment:
Should we park this "old" code in a branch somehwere? I don't think it should be merged into the main branch, as it might confuse people.

Reply:
Agreed. Just keeping it there for reference in case the latest changes aren't clear. I'll remove for the final PR.